app/views/places/show.html.erb

```diff
@@ -115,13 +115,14 @@
       		<%= f.submit "Add", class: 'btn btn-primary' %>
       	</div>
       <% end %>
-      
+     </div>
+   </div>
+</div>
 
-   <!-- Shows Photos -->
-   
-   <div class="row">
+<!-- Shows Photos -->
+   <div class="col-6">
    	<% @place.photos.each do |photo| %>
-   		<%= image_tag photo.picture %>
+   		<%= image_tag photo.picture, class: 'img-fluid' %>
```

Comment:
Adding a `class` of `img-fluid` makes the image take up no bigger space than the containing div (which I setup to be 6/12th of the screen).  This just makes your app look a little better.